README.md

```diff
@@ -176,6 +176,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+To build an application for macOS, run
+
+```sh
+make app
+```
```

Comment:
`cp -r target/release/osx/Alacritty.app /Applications/Alacritty.app`

as outlined in https://github.com/jwilm/alacritty/wiki/MacOS-application-bundle